### ISSUE TITLE
tasks got cleared without triggering a 'clear task' action

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#6140]
### Observed behavior

Observed that my list of tasks got cleared after initiating a new import

### Expected behavior

tasks should not be cleared until explicitly done by the user

### User-facing consequences

loss of data: historical context

### Errors and logs

none

### Steps to reproduce

see notes below

### Context

0.13.0 beta 1

**Comment by radinamatic** [comment]
Seeing this on beta4 too, and cannot figure out when it happens exactly: just realized that navigating back to the task manager page all previous tasks (completed *and* cancelled) disappeared without me clearing them intentionally 😕

**Comment by radinamatic** [comment]
@indirectlylit Are the tasks supposed to survive the device restart? 
I also realized that when I restart the VM all the past tasks disappear.

**Comment by indirectlylit** [comment]
> Are the tasks supposed to survive the device restart?

I don't believe so, no. That would probably be a new feature

**Comment by radinamatic** [comment]
Just making sure, not saying they should, thanks!

**Comment by rtibbles** [comment]
Correct - currently the task scheduler gets cleared at every restart.

**Comment by indirectlylit** [comment]
opened a new issue for that here: https://github.com/learningequality/kolibri/issues/6255

**Comment by indirectlylit** [comment]
There are three places I see in the code where `deleteFinishedTasks` is being inappropriately called:

```
kolibri/plugins/device/assets/src/modules/manageContent/actions/taskActions.js:
   16        () => {
   17          cancelWatch();
   18:         TaskResource.deleteFinishedTasks().then(resolve);
   19        }
   20      );

kolibri/plugins/device/assets/src/modules/wizard/utils.js:
   57        const { taskId, cancelled } = completedTask;
   58        if (taskId && !cancelled) {
   59:         return TaskResource.deleteFinishedTasks().then(() => {
   60            return getChannelWithContentSizes(transferredChannel.id);
   61          });

kolibri/plugins/facility/assets/src/modules/manageCSV/actions.js:
   67        if (task.status === TaskStatuses.COMPLETED) {
   68          store.commit(commitFinish, new Date());
   69:         TaskResource.deleteFinishedTasks();
   70        }
   71      }
```

**Comment by rtibbles** [comment]
Seems like these should be corrected to only ever delete the task in question.

**Comment by indirectlylit** [comment]
agreed

**Comment by rtibbles** [comment]
Fixed in #6355
